pkgs/by-name/co/commitlint-rs/package.nix

```
@@ -29,7 +29,7 @@ rustPlatform.buildRustPackage rec {
   meta = {
     description = "Lint commit messages with conventional commit messages";
     homepage = "https://keisukeyamashita.github.io/commitlint-rs";
-    changelog = "https://github.com/KeisukeYamashita/commitlint-rs/releases/tag/${src.rev}";
+    changelog = "https://github.com/KeisukeYamashita/commitlint-rs/releases/tag/v${version}";
```

```suggestion
    changelog = "https://github.com/KeisukeYamashita/commitlint-rs/releases/tag/${src.tag}";
```
or `src.rev` if you want. Otherwise this will 404 if the tag format changes.

---

If you're worried about that use `src.rev` which is always available.
But the biggest pitfall is the tag format changing.

---

Note that this expression doesn't even use `finalAttrs`.

---

to my understanding nix-update migrates to `src.tag`, which is a slow mass-migration in the other direction of this and the prior PR. I think we should come to a treewide preference on this one.

---

Anyway, since there is no consensus this PR has no right to make a treewide change.

---

> I agree and I have strong preference for something that doesn't require changing the changelog URL whenever the tag format changes, especially since @r-ryantm does the latter automatically.

Unsurprisingly I also agree and want a treewide preference. The tag format changing happens far less than I expected.

Maybe the point of leverage is the update script and a test inside of that, at the time that the script was run, the change log did not 404.